src/charging-station/ocpp/1.6/OCPP16IncomingRequestService.ts

```diff
@@ -867,15 +884,24 @@ export class OCPP16IncomingRequestService extends OCPPIncomingRequestService {
           ) === true
         ) {
           connectorStatus.transactionRemoteStarted = true;
+          const startTransactionData: JsonType = {
```

Comment:
```suggestion
          const startTransactionPayload: JsonType = {
```

And you can strong type to the exact payload type for the OCPP request.